lib/format_parser/version.rb

```diff
@@ -1,3 +1,3 @@
 module FormatParser
-  VERSION = '2.4.1'
```

Comment:
Just be aware that there is another PR in the works, which _might_ be merged before this one ✌️